File: Iniciante/1149 - Somando Inteiros Consecutivos/JavaScript/1149.js
function main() {
    const input = require('fs').readFileSync(0, 'utf-8').trim().split(/\s+/).map(Number);

    let a = input[0];
    let n = input[1];
    let index = 2;

    while (n <= 0) {
        n = input[index];
        index++;
    }

    let sumTotal = 0;

    for (let i = 0; i < n; i++) {
        sumTotal += a + i;
    }

    console.log(sumTotal);
}

main();

File: Iniciante/1149 - Somando Inteiros Consecutivos/Python/1149.py
def main():
    values = list(map(int, input().split()))

    a = values[0]

    for v in values[1:]:
        if v > 0:
            n = v
            break

    sum_total = 0

    for i in range(n):
        sum_total += a + i

    print(sum_total)


if __name__ == "__main__":
    main()